### DISCUSSION TITLE
zapret и его влияние на пинг в играх

### DISCUSSION THREAD

**Discussion by kiiroless** [bol-van/zapret#911] [Q&A]
помню была версия запрета для обхода блокировки дискорда но она очень и очень плохо влияла на пинг в играх вопрос в том как эта версия запрет влияет на него

**Comment by electrifying** [comment]
Запустите нужную вам игру с запретом и без и сравните пинг

**Comment by bol-van** [comment]
была не версия, а плохая таблетка
устраивающая трэш флуд
это разнесли и бездумные копипастеры потом ловили глюки
